solidity/contracts/integrator/AbstractTBTCDepositor.sol

```diff
@@ -180,6 +178,9 @@ abstract contract AbstractTBTCDepositor {
     ///        (in the context of this contract) yet.
     ///      - The deposit must be finalized on the Bridge side. That means the
     ///        deposit must be either swept or optimistically minted.
+    /// @dev This function doesn't validate if a deposit has been finalized before,
+    ///      it is a responsibility of the implementing contract to ensure this
+    ///      function won't be called twice for the same deposit.
```

Comment:
I would CAPSLOCK it.

Reply:
https://github.com/keep-network/tbtc-v2/pull/784